### PR TITLE
refactor(torque.py): remove explicit time out values for execute_wait

### PR DESCRIPTION
**Pull request by macintoshpie**
Remove wait times from torque provider, thus using the base wait time.
Addresses issue #755

### CODE REVIEW

**Comment by benclifford** [comment]
`execute_wait` doesn't seem to have a very clearly defined calling interface, which makes me think this change will not work with all channels - walltime doesn't seem to be optional on `local`.

I'm not sure which channels @macintoshpie tested this with - can you comment?

```
parsl/channels/base.py:    def execute_wait(self, cmd, walltime, envs={}, *args, **kwargs):
parsl/channels/local/local.py:    def execute_wait(self, cmd, walltime, envs={}):
parsl/channels/ssh/ssh.py:    def execute_wait(self, cmd, walltime=2, envs={}):
```

It probably makes sense to say that walltime can be optional as part of the definition of the base channel definition, and modify `local` accordingly.

**Comment by benclifford** [comment]
I'll make a separate PR to tidy up the execute_wait interface, and then I think this PR will fit cleanly on top of that.

**Comment by macintoshpie** [comment]
Yes, sorry, I hadn't tested everything yet. I had to update the call to super(). I followed the same usage/patters as in the slurm provider and it's working for me now.

File: parsl/providers/torque/torque.py [hunk 1]
```diff
@@ -76,7 +76,8 @@ def __init__(self,
                  max_blocks=100,
                  parallelism=1,
                  launcher=AprunLauncher(),
-                 walltime="00:20:00"):
+                 walltime="00:20:00",
+                 cmd_timeout=10):
```
Comment (benclifford) [inline parsl/providers/torque/torque.py]: my gut says this should be more like 30 seconds or a minute: the aim is to catch failures where the commands hang for a long long time, and I've definitely seem some queue systems take a very long time.

Comment (macintoshpie) [reply parsl/providers/torque/torque.py]: I agree, I actually just had to update my instance's to 60 seconds to not time out

Comment (benclifford) [reply parsl/providers/torque/torque.py]: in that case, make it even higher than 60 seconds by default -- the default (in my opinion) should be a decent amount (twice?) the time people are experiencing on reasonable systems.

**Pull request merged**